flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaWriterITCase.java

```diff
@@ -77,6 +77,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the standalone KafkaWriter. */
+@Testcontainers
 @ExtendWith(TestLoggerExtension.class)
 public class KafkaWriterITCase {
```

Comment:
```suggestion
class KafkaWriterITCase {
```
could be package private